packages/lib/src/index.ts

```diff
@@ -80,14 +73,29 @@ export function isMediaOutput(output: Output): output is MediaOutput {
   return "uri" in output
 }
 
+export function isMediaHosted(output: Output): output is MediaOutput {
```

Comment:
this type guard doesn't really do anything anymore. i would just delete it and check `!!output.url` to see if something is hosted